<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QWidget" name="OptionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionsWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Path to local \dbdev2Data3 drive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_pathtolocalmodels">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_browse_localmodels">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Path to local visual-mm repository</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_pathtolocalrepo">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_browse_localrepo">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>OptionsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
